src/components/elementView/detailedView/assetView/assetView.tsx

```diff
@@ -34,24 +29,23 @@ export interface AssetViewProps {
 
 // var console: any = {};
 // console.log = function() {};
-async function getData(assetkey: string, token: string) {
+function getData(assetkey: string, token: string) {
```

Comment:
why this no longer async?

Reply:
I don't think we need to use async since we are waiting for the promise to return (using `.then()` to get the response information). I haven't used await/async for any other axios requests, it was just something i played around with in the beginning

Reply:
cool